### PR TITLE
Updated compose

### PR DESCRIPTION
**Pull request by stephenwf**
@giorgiosironi

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Build shows it needs some updates the code due to related articles, maybe they are in some other branch. By the way we wanted to update bus-sdk-php, but it's not updated in the composer.lock, is it already the latest version?

**Comment by giorgiosironi** [comment]
Yeah, `elife/bus-sdk` is at 96b6df94a233704e74983a95cade04cdb9b28ed5 which is old, should get to fb9993bb0f8d5d66f94b78ae113ef89da36f142e

File: composer.lock [hunk 1]
```diff
@@ -1083,12 +1083,12 @@
...
                 "url": "https://github.com/elifesciences/bus-sdk-php.git",
-                "reference": "96b6df94a233704e74983a95cade04cdb9b28ed5"
+                "reference": "fb9993bb0f8d5d66f94b78ae113ef89da36f142e"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/elifesciences/bus-sdk-php/zipball/96b6df94a233704e74983a95cade04cdb9b28ed5",
-                "reference": "96b6df94a233704e74983a95cade04cdb9b28ed5",
+                "url": "https://api.github.com/repos/elifesciences/bus-sdk-php/zipball/fb9993bb0f8d5d66f94b78ae113ef89da36f142e",
+                "reference": "fb9993bb0f8d5d66f94b78ae113ef89da36f142e",
```
Comment (stephenwf) [inline composer.lock]: Yup fb9993bb here

**Comment by stephenwf** [comment]
Closing in favour of #38

**Pull request closed without merging**